### PR TITLE
use D2L typography styles

### PR DESCRIPTION
**Pull request by Surekh**
I think it is better and simple to use d2l-typography styles - http://design.d2l/style/typography/.
Confirmed with Erin that heading is h2 with d2l Heading 3 - Lato Bold 20 style.

### CODE REVIEW

File: components/d2l-quick-eval/activities-list/d2l-quick-eval-activities-list.js [hunk 1]
```diff
@@ -26,21 +27,10 @@ class D2LQuickEvalActivitiesList extends QuickEvalLocalize(PolymerElement) {
...
 				border-top: 1px solid var(--d2l-color-mica);
 			}
-			h2 {
-				margin-bottom: .6rem;
-				margin-top: .9rem;
-				min-height: .6rem;
-				line-height: .6rem;
-				font-size: .8rem;
+			.d2l-quick-eval-activities-course-name-heading {
+				@apply --d2l-heading-3;
```
Comment (AlexBedley) [inline components/d2l-quick-eval/activities-list/d2l-quick-eval-activities-list.js]: Hmm should this be `--d2l-heading-2`? Or should we change the html to `<h2>`? Or are the CSS and the HTML actually disjoint and the name `--d2l-heading-2` is just a suggestion?

File: components/d2l-quick-eval/activities-list/d2l-quick-eval-activities-list.js [hunk 2]
```diff
@@ -26,21 +27,10 @@ class D2LQuickEvalActivitiesList extends QuickEvalLocalize(PolymerElement) {
...
 			}
 			@media (min-width: 525px) {
-				h2 {
```
Comment (AlexBedley) [inline components/d2l-quick-eval/activities-list/d2l-quick-eval-activities-list.js]: How come this isn't necessary anymore?

File: components/d2l-quick-eval/activities-list/d2l-quick-eval-activities-list.js (hunk 1 above)
Comment (Surekh) [reply components/d2l-quick-eval/activities-list/d2l-quick-eval-activities-list.js]: It should be an H2 for A11Y with H3 for styling.

> It is important to note that Heading styles are independent of semantic Heading tag order and meaning in HTML. Most of the time they will match, but they don’t have to.

It is acceptable, for example, to use the Heading 3 style on an <h2> tag, as in this example, where the Heading 3 style is used as the title of a widget which has Heading 2 styles within it. In code, this example may or may not be implemented as an actual <h3> tag, but from a visual hierarchy standpoint the styles make sense.

File: components/d2l-quick-eval/activities-list/d2l-quick-eval-activities-list.js (hunk 2 above)
Comment (Surekh) [reply components/d2l-quick-eval/activities-list/d2l-quick-eval-activities-list.js]: I think (but you all can correct me), d2l-typography does that.

**Pull request merged**